src/commands/blocks.ts

```
@@ -144,7 +146,9 @@ export const blocksCommand = define({
 	},
 	toKebab: true,
 	async run(ctx) {
-		if (ctx.values.json) {
+		// --jq implies --json
+		const useJson = ctx.values.json || ctx.values.jq != null;
```

ESLint

[error] Unsafe assignment of an `any` value.

(ts/no-unsafe-assignment)